src/cmd/initialize.go

```diff
@@ -106,7 +121,7 @@ func downloadInitPackage(initPackageName string) error {
 
 	// If the user wants to download the init-package, download it
 	if confirmDownload {
-		utils.DownloadToFile(url, config.DeployOptions.PackagePath, "")
+		utils.DownloadToFile(url, pkgConfig.DeployOpts.PackagePath, "")
 	} else {
 		// Otherwise, exit and tell the user to manually download the init-package
 		return fmt.Errorf("you must download the init package manually and place it in the current working directory")
```

Comment:
As discussed (in person-ish), this PR will set the baseline for lang file capability and we'll slowly go about utilizing the lang file as we complete future work instead of also trying to cram that into this PR.